Content.Shared/_EinsteinEngines/Revolutionary/RevolutionaryConverterSystem.cs

```diff
@@ -0,0 +1,115 @@
+using Content.Shared._EinsteinEngines.Revolutionary.Components;
+using Content.Shared.Chat;
+using Content.Shared.Dataset;
+using Content.Shared.DoAfter;
+using Content.Shared.Interaction;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Random.Helpers;
+using Content.Shared.Revolutionary.Components;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Shared._EinsteinEngines.Revolutionary;
+
+public sealed class RevolutionaryConverterSystem : EntitySystem
+{
+    private const string RevConvertSpeechBaseKey = "revolutionary-converter-speech-";
```

Comment:
Replace it with localized dataset ID
```suggestion
    private static readonly ProtoID<LocalizedDatasetPrototype> RevConvertSpeechProto = "RevolutionaryConverterSpeech";
```